inspirehep/utils/record_getter.py

```diff
@@ -75,6 +75,18 @@ def get_es_record(record_type, recid):
 
 
 @raise_record_getter_error_and_log
+def get_es_record_by_uuid(uuid):
+    pid = PersistentIdentifier.query.filter_by(
+        object_uuid=uuid).one()
+    search_conf = current_app.config['RECORDS_REST_ENDPOINTS'][pid.pid_type]
+
+    return es.get_source(
+        index=search_conf['search_index'],
+        id=str(uuid),
+        doc_type=search_conf['search_type'])
```

Comment:
We could in principle search in all indexes in one query via `_all` and avoid the trip to the DB to know the index. Not sure if it is yet so needed though.